File: pset6/mario.py
#!/usr/bin/env python3

import cs50

def main():

    while(True):
        print("Type the pyramid's height:",end="")
        height = cs50.get_int()
        if height>0 and height<24:
                break

    for i in range(1,height+1):
        print(" "*(height-i),end="")
        print("#"*i,end="")
        print("  ",end="")
        print("#"*i,end="")
        print()

if __name__ == "__main__":
    main()

File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

void block(int i );

int main(void)
{
int heigh;

do{
    printf("Heigh: ");
    heigh = get_int();
}
while(heigh<0 || heigh>23);
for(int i=1;i<=heigh;i++){
    for(int j=heigh;j>i;j--){
        printf(" ");
    }
    block(i);
    printf("  ");
    block(i);
    printf("\n");
}

}

void block(int i)
{
    for(int j=1;j<=i;j++){
        printf("#");
    }
}